src/Standards/Generic/Docs/Formatting/SpaceBeforeCastStandard.xml

```diff
@@ -0,0 +1,31 @@
+<documentation title="Space Before Casts">
```

Comment:
```suggestion
<documentation title="Space Before Cast">
```

Reply:
I prefer `Cast` as well and I already committed your suggestion. I went with `Casts` as I saw that this is what is used in `SpaceAfterCastStandard.xml`. I believe I saw a few other sniffs where in the documentation the title contained the sniff name in the plural. Going forward, I will use the singular version of the sniff name.

Reply:
As a rule of thumb - if the sniff name is plural, use plural. If the sniff name is singular, use singular ;-)